src/xrCore/Threading/TaskManager.hpp

```diff
@@ -22,7 +22,6 @@ class TaskWorker;
 
 class XRCORE_API TaskManager final
 {
-private:
```

Comment:
Please, don't removed `private` and `public` labels. These are there to break the code into logical blocks.